src/io/jesi/backpack/collection.cljc

```diff
@@ -365,3 +365,10 @@
   (if (empty? x)
     nil
     x))
+
+(defn distinct-vals?
+  "Returns true if all the map values are unique "
+  [m]
+  ;TODO this the most efficient way?
```

Comment:
Any ideas?

Reply:
`(comp (partial apply distinct?) vals)`

Reply:
https://clojuredocs.org/clojure.core/distinct_q

Reply:
Looks like the counting a set is faster. See https://github.com/jesims/backpack/pull/83/files#diff-7eb79c9927f780b45f29cfed61267af6f19f32d7ca022a28dc661cd8b0949430R885